.travis.yml

```diff
@@ -12,16 +12,18 @@ script:
 matrix:
   fast_finish: true
   include:
-  - rvm: 2.4.0
-    env: PUPPET_VERSION="~> 4.0" CHECK=syntax
-  - rvm: 2.4.0
-    env: PUPPET_VERSION="~> 4.0" CHECK=lint
-  - rvm: 2.4.0
-    env: PUPPET_VERSION="~> 4.0" CHECK=spec
-  - rvm: 2.4.0
-    env: PUPPET_VERSION="~> 4.0" CHECK=metadata_lint
-  - rvm: 2.4.0
-    env: PUPPET_VERSION="~> 4.0" CHECK=build DEPLOY_TO_FORGE=yes
+  - rvm: 2.5.1
```

Comment:
could we keep support/tests for Puppet 4.x please

Reply:
Added in 8556db2124339f131f8becb3596b417a64f4e9f6

Reply:
Build is failing.

I'm afraid that supporting the module on an unsupported version of Puppet running on an unsupported version of Ruby is a waste of resources :confused:

Reply:
To complete information:

* [The latest version of Puppet 4.x (4.10.2)  ships in PE 2016.4.15 with Ruby 2.1.9](https://puppet.com/docs/pe/2019.0/component_versions_in_recent_pe_releases.html)
* [PE 2016.4.15 has reached EOL in October 2018](https://groups.google.com/forum/#!topic/puppet-users/78ZcVVyuIMQ)
* [Ruby 2.1.x has reached EOL in March 2017](https://en.wikipedia.org/wiki/Ruby_(programming_language)#Table_of_versions)

Reply:
ah right, fair enough thanks for the info!

Reply:
Fine, removing 8556db2124339f131f8becb3596b417a64f4e9f6